src/a11y-dialog.ts

```diff
@@ -75,18 +80,14 @@ export default class A11yDialog {
   /**
    * Show the dialog element, trap the current focus within it, listen for some
    * specific key presses and fire all registered callbacks for `show` event
-   * @param {Event} event
-   * @returns {A11yDialogType}
    */
-  show = event => {
+  public show = (event?: Event): A11yDialogInstance => {
     // If the dialog is already open, abort
-    if (this.shown) {
-      return this
-    }
+    if (this.shown) return this
 
     // Keep a reference to the currently focused element to be able to restore
     // it later
-    this.previouslyFocused = document.activeElement
+    this.previouslyFocused = document.activeElement as HTMLElement
```

Comment:
`document.activeElement` is typed as `Element`, but we know better – we know this element is going to be an `HTMLElement`, and we need to assert as much so TypeScript doesn't get mad.